### ISSUE TITLE
Ubuntu 18.04 cri-o-1.14 latest PPA version crio-wipe.service and crio-wipe.bash missed on .deb package

### ISSUE THREAD

**Issue by Slach** [cri-o/cri-o#2696]
Steps to reproduce
```
apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 018BA5AD9DF57A4448F0E6CF8BECF1637AD8C79D
echo "deb http://ppa.launchpad.net/projectatomic/ppa/ubuntu bionic main" > /etc/apt/sources.list.d/crio.list
apt-get install -y cri-o-1.14
systemctl enable crio
systemctl restart crio
```

expected result - restarted crio service
actual result
```
Failed to restart crio.service: Unit crio-wipe.service not found.
```

```
# dpkg -l cri-o-1.14
1.14.10-1~dev~ubuntu18.0

# dpkg -L cri-o-1.14 | grep service
/lib/systemd/system/crio-shutdown.service
/lib/systemd/system/crio.service

# dpkg -L cri-o-1.14 | grep wipe

```

**Comment by haircommander** [comment]
Ok fine openshift bot, @lsm5 PTAL

**Comment by davidhadas** [comment]
Hi, 
can anyone suggest a workaround?

**Comment by haircommander** [comment]
`systemctl status crio` to find the location of the unit file (probably something like `/usr/lib/systemd/system/crio/crio.service` and remove the line `Requires=crio-wipe.service`

**Comment by jedblack** [comment]
There are other things missing from this package, in addition to the missing unit files, its also missing...
/etc/sysconfig/crio (thats typically a RHEL location, but should be created for Ubuntu too)
It also is missing this : /usr/lib/crio/bin/conmon

**Comment by lsm5** [comment]
should be fixed with 1.14.10-1. Check it out..

**Comment by Slach** [comment]
yes wipe added and restart fine
thanks a lot
